tests/phpunit/CRM/MembersOnlyEvent/Service/MembersOnlyEventAccessServiceTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+use CRM_MembersOnlyEvent_Service_MembersOnlyEventAccessService as MembersOnlyEventAccessService;
+use CRM_MembersOnlyEvent_Test_Fabricator_MembersOnlyEvent as MembersOnlyEventFabricator;
+use CRM_MembersOnlyEvent_Test_Fabricator_Contact as ContactFabricator;
+
+require_once __DIR__ . '/../../../BaseHeadlessTest.php';
+
+/**
+ * Class CRM_MembersOnlyEvent_BAO__MembersOnlyEventPriceFieldValue
+ *
+ * @group headless
+ */
+class CRM_MembersOnlyEvent_Service_MembersOnlyEventAccessServiceTest extends BaseHeadlessTest {
```

Comment:
same apply here, this should be named `CRM_MembersOnlyEvent_Service_MembersOnlyEventAccessTest `